flake.nix

```diff
@@ -232,11 +227,34 @@
               gotools
               go-tools
               gofumpt
-            ];
+            ]);
             shellHook = ''
               ${pre-commit.shellHook}
             '';
           };
+
+          tf = {
+            inherit (pkgs) terraform opentofu;
+          };
+        in
+        {
+          inherit terraformProviders;
+
+          packages = getPackages tf.opentofu;
+
+          checks = getChecks tf.opentofu;
+
+          lib = {
+            mkDevShell-terraform = mkDevShell tf.terraform;
+            mkDevShell-opentofu = mkDevShell tf.opentofu;
+            mkDevShell = mkDevShell tf.opentofu;
+          };
+
+          devShells = rec {
+            terraform = getDevShell tf.terraform;
+            opentofu = getDevShell tf.opentofu;
+            default = opentofu;
+          };
         }) // {
       templates = rec {
```

Comment:
I've never used opentofu; I guess it's intended to be backward compatible with Terraform code dating pre-fork. But just in case, do you think there should be anything to do on the templates and examples and if we set `opentofu` to be the default?

Reply:
from what i can see, the existing examples look simple enough i don't expect problems with them.
problematic parts could be hardcoded terraform registry URIs or syntax newer than what opentofu has caught up to, but i'm not seeing us use either of those yet so far.